packages/frontend/app/components/UserAvatar.tsx

```diff
@@ -41,9 +41,16 @@ export default function UserAvatar({
       {...props}
       icon={<UserOutlined />}
       src={
-        photoURL && photoURL.startsWith('http')
-          ? photoURL
-          : '/api/v1/profile/get_picture/' + photoURL
+        photoURL && photoURL.startsWith('http') ? (
+          <img src={photoURL} alt={username} loading="lazy" decoding="async" />
+        ) : (
+          <img
+            src={`/api/v1/profile/get_picture/${photoURL}`}
+            alt={username}
+            loading="lazy"
+            decoding="async"
+          />
+        )
```

Comment:
huh i didn't know you could provide an img element. What if we used Next.js's Image tag? Since I believe it does some fancy caching things that could be nice maybe.

Reply:
I tried it out and sadly, it messes with the API call for the image from the backend so I'll keep it as is for now.